lib/apache/ant-zipscanner/README

```diff
@@ -0,0 +1,17 @@
+To rebuild ant-zipscanner, follow those steps, in a temporary directory:
+
+    unzip $GWT_TOOLS/lib/apache/ant-1.6.5-src.zip
+
+    patch -p0 --binary -i $GWT_TOOLS/lib/apache/ant-zipscanner-1.6.5.patch
+    # the --binary argument is necessary due to CRLF line endings
+
+    javac org/apache/tools/ant/types/ZipScanner.java
+    # this assumes JDK 8; if using JDK 9+, pass an additional --release 8
+
+    jar -Mcvf $GWT_TOOLS/lib/apache/ant-zipscanner/ant-zipscanner-1.6.5.jar \
```

Comment:
consider a -1 suffix in case we find we did something silly and need to make another one also based on 1.6.5?

Reply:
Done